src/utility/clearBody.js

```diff
@@ -0,0 +1,9 @@
+export const clearBody = () => {
```

Comment:
@Mmarvinn Pls rename folder from `utility` to `utils`

Reply:
fix